examples/robolectricExample/build.gradle

```diff
@@ -0,0 +1,51 @@
+buildscript {
+    repositories {
+        jcenter()
+    }
+    dependencies {
+        classpath 'de.undercouch:gradle-download-task:2.0.0'
+    }
+}
+
+apply plugin: 'de.undercouch.download'
+apply plugin: 'android-sdk-manager'
+apply plugin: 'com.android.application'
+apply plugin: 'android-command'
+apply plugin: 'com.neenbedankt.android-apt'
+apply plugin: 'realm-android'
+apply plugin: 'realm-robolectric'
```

Comment:
Well the plugin will hopefully work for both Windows and Linux soon :), but I get your point. It is probably only a minority that wants to use Robolectrics.